README.md

```diff
@@ -39,6 +39,11 @@ Consult the [Roadmap](/ROADMAP.md) for a complete state description of the proje
   - [HTTP-API](#http-api)
   - [IPFS Core examples (use IPFS as a module)](#ipfs-core-examples-use-ipfs-as-a-module)
     - [Create a IPFS node instance](#create-a-ipfs-node-instance)
+    - [Find IPFS in Node.js](#find-ipfs-in-nodejs)
+    - [Find IPFS in the Browser](#find-ipfs-in-the-browser)
```

Comment:
same goes for this one:

> I don't believe this will be needed as with the examples we have right now on https://github.com/ipfs/js-ipfs/tree/master/examples, but feel welcome to complement them.